1_data/src/subset_nwm.R

```diff
@@ -99,7 +99,27 @@ subset_nwm <- function(ind_file, model_configuration, comids, gd_config) {
     }
   }
 
-  ncvar_put(new_nc, new_nc$var$streamflow, streamflow)
+  # need to loop through the sites because the dimensions in streamflow matrix were written incorrectly
```

Comment:
I'm not really following what is meant by "written incorrectly". We should be able to construct an array that can be written all in one go with ncvar_put right?